config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do  
+  get 'rooms/new' => 'rooms#new'
```

Comment:
Please change this to `resources :rooms`.
Don't forget to update the link in the header as well after checking `rails routes` in the terminal.